testsuite/podman_runner/12_run_salt_sle_minion.sh

```diff
@@ -2,7 +2,11 @@
 set -xe
 src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
 
-podman run --rm -d --network uyuni-network-1 -v /tmp/test-all-in-one:/tmp -v ${src_dir}/testsuite/podman_runner/salt-minion-entry-point.sh:/salt-minion-entry-point.sh --name sle_minion -h sle_minion ghcr.io/$UYUNI_PROJECT/uyuni/ci-test-opensuse-minion:$UYUNI_VERSION bash -c "/salt-minion-entry-point.sh uyuni-server-all-in-one-test 1-SUSE-KEY-x86_64"
+echo opensuseminionproductuuid > /tmp/opensuse_product_uuid
+
+podman run --privileged --rm -d --network uyuni-network-1 -v /tmp/opensuse_product_uuid:/sys/class/dmi/id/product_uuid -v /tmp/test-all-in-one:/tmp -v ${src_dir}/testsuite/podman_runner/salt-minion-entry-point.sh:/salt-minion-entry-point.sh --name sle_minion -h sle_minion ghcr.io/$UYUNI_PROJECT/uyuni/ci-test-opensuse-minion:$UYUNI_VERSION bash -c "/salt-minion-entry-point.sh uyuni-server-all-in-one-test 1-SUSE-KEY-x86_64"
 podman exec sle_minion bash -c "ssh-keygen -A && /usr/sbin/sshd -e"
 podman exec sle_minion bash -c "if [ ! -d /root/.ssh ];then mkdir /root/.ssh/;chmod 700 /root/.ssh;fi;cp /tmp/authorized_keys /root/.ssh/"
 
+podman exec sle_minion bash -c "echo DEBUG;cat /sys/class/dmi/id/product_uuid;dmidecode | grep 4c4c4544-0037-3710-8047-c7c04f353632;echo FI_DEBUG"
```

Comment:
Don't we need to generate a new one each time? 🤔

Reply:
Yep. I have removed this line from the code.